ProjFS.Mac/PrjFSKext/VnodeCache.cpp

```diff
@@ -428,3 +506,30 @@ KEXT_STATIC bool TryInsertOrUpdateEntry_ExclusiveLocked(
     
     return false;
 }
+
+KEXT_STATIC_INLINE void InitCacheStats()
```

Comment:
Nit: do we need atomic access here during initialization, or can this be a memeset? Definitely does not block the PR, just curious.

Reply:
> Nit: do we need atomic access here during initialization, or can this be a memeset? Definitely does not block the PR, just curious.

Things get tricky because these are not plain old datatypes.

When running the unit tests these variables are `std::atomic<T>`:

```
#ifdef KEXT_UNIT_TESTING
#include <atomic>
#define _Atomic(X) std::atomic< X >
```

And in general memsetting STL data structures is not good (and as discussed in a separate comment, we could use `=` here, but that would be no different than calling `atomic_store` ourselves).

I would not be surprised if there's a risk from memsetting [C atomic types](https://en.cppreference.com/w/c/language/atomic), but because this is the initialization path I'd rather take whatever small perf hit (if any) comes from using `atomic_store`